README.md

```diff
@@ -228,6 +228,10 @@ with a remote `<root_uri>`.
 
 Running semantic segmentation on roads trains a Mobilenet for 100k steps which takes about 6hrs on a P3 instance.
 
+#### Experimental: Use vector tiles to get labels
+
+It is possible to use a Mapbox vector tile endpoint (which contains `{z}/{x}/{y}` as a source of labels. We have tested this with a set of vector tiles created from Open Street Map (OSM). Note that most buildings and some roads present in the Spacenet labels are not present in the OSM data. The `class_id_to_filter` values in the experiment config might need to be changed if you are not using OSM. To use vector tiles instead of GeoJSON, run the experiment with the following flags: `-a mbtiles_uri <URI> -a mbtiles_zoom <ZOOM_LEVEL>`. 
```

Comment:
Perhaps a small bit of text explaining what `class_id_to_filter` does would be helpful to the reader.  Aside from that +1.

Reply:
Fixed.